workers_test.go

```diff
@@ -818,6 +828,28 @@ func TestWorkers_ListWorkerRoutesEnt(t *testing.T) {
 	}
 }
 
+func TestWorkers_GetWorkerRoute(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/zones/foo/workers/routes/1234", func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodGet, r.Method, "Expected method 'GET', got %s", r.Method)
+		w.Header().Set("content-type", "application-json")
```

Comment:
this is an invalid content-type according to [rfc7231 § 3.1.1.1](https://tools.ietf.org/html/rfc7231#section-3.1.1.1) but it's somehow working 🤷‍♂️